timeseries/tests/unittests/test_predictor.py

```diff
@@ -1034,26 +1036,32 @@ def test_when_log_to_file_set_then_predictor_logs_to_file(temp_model_path):
 
 def test_when_log_file_set_then_predictor_logs_to_custom_file(temp_model_path):
     predictor = TimeSeriesPredictor(path=temp_model_path, log_to_file=True, log_file_path="custom_log.txt")
-    predictor.fit(DUMMY_TS_DATAFRAME, hyperparameters={"Naive": {}})
     log_path = Path(".") / "custom_log.txt"
-    assert Path.exists(log_path)
+    try:
+        predictor.fit(DUMMY_TS_DATAFRAME, hyperparameters={"Naive": {}})
+        assert Path.exists(log_path)
 
-    # check if the log contains text
-    with open(log_path, "r") as f:
-        log_text = f.read()
-    assert "Naive" in log_text
+        # check if the log contains text
+        with open(log_path, "r") as f:
+            log_text = f.read()
+        assert "Naive" in log_text
+    finally:
+        log_path.unlink(missing_ok=True)
 
 
 def test_when_log_file_set_with_pathlib_then_predictor_logs_to_custom_file(temp_model_path):
     predictor = TimeSeriesPredictor(path=temp_model_path, log_to_file=True, log_file_path=Path(".") / "custom_log.txt")
-    predictor.fit(DUMMY_TS_DATAFRAME, hyperparameters={"Naive": {}})
     log_path = Path(".") / "custom_log.txt"
-    assert Path.exists(log_path)
+    try:
+        predictor.fit(DUMMY_TS_DATAFRAME, hyperparameters={"Naive": {}})
+        assert Path.exists(log_path)
 
-    # check if the log contains text
-    with open(log_path, "r") as f:
-        log_text = f.read()
-    assert "Naive" in log_text
+        # check if the log contains text
+        with open(log_path, "r") as f:
+            log_text = f.read()
+        assert "Naive" in log_text
+    finally:
+        log_path.unlink(missing_ok=True)
```

Comment:
Why is this change necessary?

Reply:
The test left the `custom_log.txt` file in the local folder. This makes the test cleans up after itself.